templates/index.html

```diff
@@ -43,7 +43,7 @@ <h1 class="site-section-meta border-bottom">Latest News</h1>
                 <h3 class="mb-0">{{ post.title }}</h3>
                 <div class="mb-1 text-muted">{{ post.date }}</div>
                 <p class="card-text">{{ post.content | safe | truncate(200)}}</p>
-                <a href="/news" class="card-link">Continue reading</a>
```

Comment:
hopefully this isn't needed after the fix to the root URL on breadboard (`/news' and `news' should point to the same place)